lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -384,11 +384,11 @@ public function getListSequencesSQL($database)
     /**
      * {@inheritDoc}
      */
-    protected function _getCreateTableSQL($table, array $columns, array $options = [])
+    protected function _getCreateTableSQL($tableName, array $columns, array $options = [])
```

Comment:
This is against the approach taken in https://github.com/doctrine/dbal/pull/4172:
> Argument names have been simplified where possible (e.g. `dropTable($tableName)` → `dropTable($name)`).

Unless we're dealing both with a table and a name in the same context, there's no much point in being too specific IMO.